ecs-init/docker/docker.go

```diff
@@ -440,6 +463,33 @@ func (c *client) getHostConfig(envVarsFromFiles map[string]string) *godocker.Hos
 	return createHostConfig(binds)
 }
 
+// getCredentialsFetcherSocketBind returns the bind for CredentialsFetcher socket and also whether the bind exists.
+// Value for the bind is as follow:
+// CREDENTIALS_FETCHER_HOST (as in os.Getenv) set: source CREDENTIALS_FETCHER_HOST (as in os.Getenv, trim unix:// prefix),
```

Comment:
I am having trouble understanding this comment. What does this mean? Can you rephrase it?

Reply:
did rephrase